staging/src/k8s.io/component-base/version/verflag/verflag.go

```diff
@@ -60,27 +58,24 @@ func (v *versionValue) Set(s string) error {
 	return err
 }
 
-func (v *versionValue) String() string {
-	if *v == VersionRaw {
-		return strRawVersion
-	}
-	return fmt.Sprintf("%v", bool(*v == VersionTrue))
+func (v *VersionValue) String() string {
+	return string(*v)
 }
 
 // The type of the flag as required by the pflag.Value interface
-func (v *versionValue) Type() string {
+func (v *VersionValue) Type() string {
 	return "version"
 }
 
-func VersionVar(p *versionValue, name string, value versionValue, usage string) {
+func VersionVar(p *VersionValue, name string, value VersionValue, usage string) {
```

Comment:
Just a note: This helper should probably accept an arbitrary flagset, rather than forcing use of the global flagset.
We could add a new helper if changing this one would break compatibility with existing consumers, and we can address this in a future PR, but something to keep in mind.

Reply:
K, I'd probably add a new method for that